using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -647,7 +651,9 @@ ACPIdentity.getUrlVariables().then(urlVariables => console.log("AdobeExperenceSD
 {% tab title="Android" %}
 ### getIdentifiers <a id="getIdentifiers-java"></a>
 
-This API returns all customer identifiers that were previously synced with the Adobe Experience Cloud through the [AdobeCallback](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/identity/identity-api-reference#adobecallback).
+This API returns all customer identifiers that were previously synced with the Adobe Experience Cloud through the _AdobeCallback_.
+
+When `AdobeCallbackWithError` is provided, a 500ms timeout is used when fetching the custom identifiers from the Mobile SDK. If the operation times out or an unexpected error occurs, the `fail` method is called with the appropriate `AdobeError`.
```

Comment:
Please reword as follows:
"When `AdobeCallbackWithError` is provided, and you are fetching the custom identifiers from the Mobile SDK, the timeout value is 500ms. If the operation times out or an unexpected error occurs, the `fail` method is called with the appropriate `AdobeError`."